### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MeNiks**

### CODE REVIEW

**Review by nokite — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Enhanced WebView component for Android that works as intended out of the box
 
      ```gradle
      dependencies {
-         compile 'com.github.delight-im:Android-AdvancedWebView:v3.0.0'
+         implementation("com.github.delight-im:Android-AdvancedWebView:v3.2.0")
```
Comment (nokite) [inline README.md:26]: Nice bumping the version and using implementation. I don't think we need the braces though.

Comment (MeNiks) [reply README.md:26]: No braces are required. I tried your input in kotlin DSL but the gradle failed. Above is correct

Comment (nokite) [reply README.md:26]: The readme does not suggest that Kotlin DSL is used. I'd say either mention it, or use the more conventional syntax.

Comment (MeNiks) [reply README.md:26]: The above syntax is common for both gradle & kotlin DSL

**Comment by ocram** [comment]
Thank you!

Merged in https://github.com/delight-im/Android-AdvancedWebView/commit/51d1850d119bd73d10b0eda52813b99e3c56cf59 and https://github.com/delight-im/Android-AdvancedWebView/commit/68aa3ca352988c12baeeb0f9bf304576c087690c.

**Pull request closed without merging**